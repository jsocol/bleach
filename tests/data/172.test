<div id="103"><script>history.pushState(0,0,'/i/am/somewhere_else');</script>//["'`-->]]>]</div><div id="104"><svg xmlns="http://www.w3.org/2000/svg" id="foo">
<x xmlns="http://www.w3.org/2001/xml-events" event="load" observer="foo" handler="data:image/svg+xml,%3Csvg%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%3E%0A%3Chandler%20xml%3Aid%3D%22bar%22%20type%3D%22application%2Fecmascript%22%3E alert(104) %3C%2Fhandler%3E%0A%3C%2Fsvg%3E%0A#bar"/>
</svg>//["'`-->]]>]</div>
