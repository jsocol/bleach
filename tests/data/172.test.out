<div id="103">//["'`--&gt;]]&gt;]</div><div id="104"><svg id="foo" xmlns="http://www.w3.org/2000/svg">

</svg>//["'`--&gt;]]&gt;]</div>
